net-nntp/sabnzbd/sabnzbd-4.2.2.ebuild

```diff
@@ -0,0 +1,164 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{10..12} )
+PYTHON_REQ_USE="sqlite"
+
+inherit optfeature python-single-r1 systemd
+
+MY_PV="${PV/_alpha/Alpha}"
+MY_PV="${MY_PV/_beta/Beta}"
+MY_PV="${MY_PV/_rc/RC}"
+
+MY_P="SABnzbd-${MY_PV}"
+
+DESCRIPTION="Binary newsgrabber with web-interface"
+HOMEPAGE="https://sabnzbd.org/"
+SRC_URI="https://github.com/sabnzbd/sabnzbd/releases/download/${MY_PV}/${MY_P}-src.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+# Sabnzbd is GPL-2 but bundles software with the following licenses.
+LICENSE="GPL-2 BSD LGPL-2 MIT BSD-1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	acct-user/sabnzbd
+	acct-group/sabnzbd
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/chardet[${PYTHON_USEDEP}]
+		dev-python/cheetah3[${PYTHON_USEDEP}]
+		dev-python/cherrypy[${PYTHON_USEDEP}]
+		dev-python/configobj[${PYTHON_USEDEP}]
+		dev-python/cryptography[${PYTHON_USEDEP}]
+		>=dev-python/feedparser-6.0.11[${PYTHON_USEDEP}]
+		>=dev-python/guessit-3.7.1[${PYTHON_USEDEP}]
+		dev-python/notify2[${PYTHON_USEDEP}]
+		dev-python/portend[${PYTHON_USEDEP}]
+		dev-python/puremagic[${PYTHON_USEDEP}]
+		~dev-python/sabctools-8.1.0[${PYTHON_USEDEP}]
+	')
+"
+RDEPEND="
+	${DEPEND}
+	|| (
+		>=app-arch/par2cmdline-0.8
+		>=app-arch/par2cmdline-turbo-1.1.0
+	)
+	net-misc/wget
+"
+BDEPEND="
+	test? (
+		$(python_gen_cond_dep '
+			dev-python/flaky[${PYTHON_USEDEP}]
+			>=dev-python/lxml-4.5.0[${PYTHON_USEDEP}]
+			dev-python/pyfakefs[${PYTHON_USEDEP}]
+			dev-python/pytest-httpbin[${PYTHON_USEDEP}]
+			dev-python/pytest-httpserver[${PYTHON_USEDEP}]
+			dev-python/pytest[${PYTHON_USEDEP}]
+			dev-python/requests[${PYTHON_USEDEP}]
+			dev-python/selenium[${PYTHON_USEDEP}]
+			dev-python/tavalidate[${PYTHON_USEDEP}]
+			>=dev-python/tavern-2[${PYTHON_USEDEP}]
+			dev-python/werkzeug[${PYTHON_USEDEP}]
+			dev-python/xmltodict[${PYTHON_USEDEP}]
+		')
+		app-arch/p7zip
+		app-arch/unrar
+		app-arch/unzip
+		www-apps/chromedriver-bin
+	)
+"
+
+src_test() {
+	local EPYTEST_IGNORE=(
+		# network sandbox
+		tests/test_getipaddress.py
+		tests/test_rss.py
+		tests/test_urlgrabber.py
+		tests/test_utils/test_happyeyeballs.py
+		tests/test_utils/test_internetspeed.py
+	)
+	local EPYTEST_DESELECT=(
+		# network sandbox
+		'tests/test_cfg.py::TestValidators::test_validate_host'
+		'tests/test_consistency.py::TestWiki'
+		'tests/test_newswrapper.py::TestNewsWrapper'
+		'tests/test_happyeyeballs.py::TestHappyEyeballs'
+		'tests/test_internetspeed.py::TestInternetSpeed'
+		# Doesn't work, complains mocker missing even when pytest-mock installed
+		'tests/test_dirscanner.py::TestDirScanner'
```

Comment:
Did you add `-p pytest_mock`, since you're disabling autoloading?

Reply:
still fails, new error though: fixture 'fs' not found

Will add the testdep for pytest-mock.